SparkPost/2web2ui
# FE-737: Mobile Nav Closes on Click of NavItem

### What Changed
 - The mobile navigation will now properly close when you click on a nav item

### How To Test
 - Shrink window down to mobile size
 - Open the navigation
 - Toggle a NavGroup
   - Ensure this does not close the navigation
 - Click on an individual NavItem
   - Ensure this properly closes the navigation
